model/config.go

```diff
@@ -453,6 +454,10 @@ func (s *ServiceSettings) SetDefaults(isUpdate bool) {
 		s.EnableOutgoingWebhooks = NewBool(true)
 	}
 
+	if s.IncomingWebhooksTriggerOutgoingWebhooks == nil {
+		s.IncomingWebhooksTriggerOutgoingWebhooks = NewBool(true)
```

Comment:
I think the default should probably be the previous behaviour.  @mickmister?

Reply:
You're probably right. Fixed it.